bridge/src/observers/nine-chronicles.ts

```diff
@@ -117,24 +144,14 @@ export class NCGTransferredEventObserver implements IObserver<{ blockHash: Block
                 } else if(blockHash === "4a3e74ffb45a317085c5bde76159069db42ff04d45f62d64cbac4ac1e5ec503c") {
                   transactionHash = "0xc60146f55fd24323de2e2efbd66743317ac9601aa648daf9d78069661635c306";
                 } else {
-                  const { transactionHash: txHash } = await this._wrappedNcgTransfer.mint(recipient, ethereumExchangeAmount);
+                  const { transactionHash: txHash } = await this._wrappedNcgTransfer.mint(recipient!, ethereumExchangeAmount);
```

Comment:
Note: The reason I used `!` here is TypeScript static code analysis doesn't support this.

```TypeScript
a: string | null
if (a === null) {
  continue;
}

a // 'string' type

// But

a: string | null

const isNull = a === null
if (isNull) {
  continue;
}

a // 'string | null' type.
```

It expects it is `string` type, non-nullable, through `isInvalidTx` check.

```
if (isInvalidTx) {
    const nineChroniclesTxId = await this._ncgTransfer.transfer(sender, amountString, "I'm bridge and you should transfer with memo, valid ethereum address to receive.");
    console.log("Valid memo doesn't exist so refund NCG. The transaction's id is", nineChroniclesTxId);
    continue;
                    
```